src/util/Constants.js

```diff
@@ -34,6 +52,9 @@ const { Error, RangeError } = require('../errors');
  * (or 0 for never)
  * @property {number} [restGlobalRateLimit=0] How many requests to allow sending per second (0 for unlimited, 50 for
  * the standard global limit used by Discord)
+ * @property {string[]|RateLimitQueueFilter} [disableRateLimitQueue] Decides how rate limits (429) should be handled.
```

Comment:
This isn't just invoked when a 429 was received, but also when djs preemptively throttles to avoid that.
This docstring doesn't really make this clear, which might be confusing.